hugegraph-dist/README.md

```diff
@@ -10,7 +10,7 @@ We can use docker to quickly start an inner HugeGraph server with RocksDB in bac
 
 2. Using docker compose
 
-    We can also use `docker-compose up -d`.  The `docker-compose.yaml` is below:
+    Additionally, if we want to manage other Hugegraph-related instances with `server` in a single file, we can also use docker-compose for deployment. (Certainly we can only deploy server without other instance) We can also use `docker-compose up -d`.  The `docker-compose.yaml` is below:
```

Comment:
- Hugegraph-related => HugeGraph-related.
- "(Certainly we can only deploy server without other instance)" -- move to the first line? like "Certainly We can deploy server without other instances. Additionally we can also ...".
- To deploy HugeGraph-related instances via `docker-compose up -d`.
- "is below" => "is as below"